docs/tutorials/01-intro.md

```diff
@@ -0,0 +1,209 @@
+# Buildah Tutorial 1
+## Building OCI container images
+
+The purpose of this tutorial is to demonstrate how buildah can be used to build container images compliant with the [Open Container Initiative](https://www.opencontainers.org/) (OCI) [image specification](https://github.com/opencontainers/image-spec). Images can be built from existing images, from scratch, and using Dockerfiles. OCI images built using the buildah command line tool (CLI) and the underlying OCI based technologies (e.g. [containers/image](https://github.com/containers/image) and [containers/storage](https://github.com/containers/storage)) are portable and can therefore run in a docker environment.
```

Comment:
@nalind, is the last bit of this true?  I thought we were still hung up wating for #187 to be fully functional in a docker environment?

Reply:
@TomSweeneyRedHat I actually demonstrate it. Now if there are other portability/compatibility issues then we can mention them but for now and for this tutorial the Buildah image can be pushed to docker-daemon and I can run it from there.

Reply:
#187's mainly about adapting to changes in containers/image that modify how it uses containers/storage to better handle reading of images.  An effect of those changes is that tools like skopeo can reliably read images from disk, making buildah's "commit" and "push" logic much simpler.  But even without that, the images we produce should be usable.

Reply:
And is.

Reply:
Hmm, I will have to play with that some more.  I thought it was broken the last time I tried and I'd put a blog on hold due to that.  Thanks for the 411.